### PR TITLE
Log any error we get back from stats

### PR DESCRIPTION
**Pull request by nickoneill**
...in an attempt to get more info on one users's weird error

### CODE REVIEW

File: FiveCalls/FiveCalls/MyImpactViewController.swift [hunk 1]
```diff
@@ -156,6 +156,9 @@ class MyImpactViewController : UITableViewController {
             // Fetch the user's call stats
             let userStatsOp = FetchUserStatsOperation()
             userStatsOp.completionBlock = {
+                if let error = userStatsOp.error {
+                    Answers.logCustomEvent(withName: "userstats-error", customAttributes: ["message": error.localizedDescription])
```
Comment (subdigital) [inline FiveCalls/FiveCalls/MyImpactViewController.swift]: Can we use Mixpanel hear too?

Comment (nickoneill) [reply FiveCalls/FiveCalls/MyImpactViewController.swift]: We can - though I feel like we're misusing either tool using it like this. Is there a generic error logging tool that would do a better job for it?

**Comment by nickoneill** [comment]
Looks amazing - I just applied. Looks like they support JS error tracking too, so we can drop sentry and unify errors in one place!

**Comment by nickoneill** [comment]
Bugsnag integrated! Looks like it's much more well suited for this. I'm going to limit our integration to this one area until we a) hear back about the open source plan and b) get some real data into the dashboard so we can evaluate how useful it is.

File: FiveCalls/FiveCalls/AppDelegate.swift [hunk 2]
```diff
@@ -62,6 +64,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
 
+    func bugsnapStartup() {
```
Comment (subdigital) [inline FiveCalls/FiveCalls/AppDelegate.swift]: bugsnap?

**Comment by subdigital** [comment]
I have similar comments about not letting bugsnag leak everywhere in every file, and use an extension method to keep that implementation detail in one place, however I hear your point about trying it out. If we end up committing to it we can move this logic.

File: FiveCalls/FiveCalls/AppDelegate.swift (hunk 2 above)
Comment (nickoneill) [reply FiveCalls/FiveCalls/AppDelegate.swift]: lol, fixed

**Pull request merged**